auth/src/main/java/com/firebase/ui/auth/util/data/AuthViewModelBase.java

```diff
@@ -0,0 +1,28 @@
+package com.firebase.ui.auth.util.data;
+
+import android.app.Application;
+
+import com.firebase.ui.auth.util.ui.FlowHolder;
+import com.firebase.ui.auth.util.ui.ViewModelBase;
+import com.google.firebase.FirebaseApp;
+import com.google.firebase.auth.FirebaseAuth;
+import com.google.firebase.auth.PhoneAuthProvider;
+
+public class AuthViewModelBase extends ViewModelBase<FlowHolder> {
+    protected FlowHolder mFlowHolder;
```

Comment:
As in my other comment, would prefer to make each of these private and then access in child classes through getter.

Reply:
Done.